crates/services/p2p/src/service.rs

```diff
@@ -756,6 +849,9 @@ where
                     Some(FuelP2PEvent::InboundRequestMessage { request_message, request_id }) => {
                         self.process_request(request_message, request_id)?
                     },
+                    Some(FuelP2PEvent::NewSubscription { peer_id, .. }) => {
+                        let _ = self.broadcast.new_tx_subscription_broadcast(FuelPeerId::from(peer_id.to_bytes()));
+                    },
```

Comment:
This code currently works for all possible subscribers. I think we just want to know about subscribers for transactions. I think we either check the tag of subscription, or pass the tag along with peer id.

Reply:
Done